frontend/src/components/bucket/BucketList.vue

```diff
@@ -50,6 +58,12 @@ onMounted(async () => {
     <div>
       <h1>Select a bucket</h1>
       <h3>Buckets are containers for storing objects.</h3>
+      <Message
+        v-if="getConfig?.notifications?.homeBanner"
+        severity="warn"
+      >
+        {{ getConfig?.notifications?.homeBanner }}
+      </Message>
```

Comment:
Do we want to be mounting here, or would it be a better idea to make this feature more generic and mounted higher up the views/router tree?

Reply:
Yeah I think there could be a "top level" notification that appears above the header in the global layout. But this one was for the spots defined by Sharolyn in the ticket. We could add other notifications in this context as well.

Reply:
I could see us inserting this into `bcbox/AppLayout.vue` between the header and main views slots...
```vue
    <!-- Header/Nav -->
    <div class="layout-head">
      <Header />
      <slot name="nav" />
    </div>

    <!-- Insert the "Global Notification Component" here -->

    <!-- Main views -->
    <main class="layout-container">
      <slot name="main" />
    </main>
```

Reply:
To be considered at a later date 👍